src/data/set/lattice.lean

```diff
@@ -77,10 +77,16 @@ notation `⋂` binders `, ` r:(scoped f, Inter f) := r
 ⟨λ ⟨t, ⟨⟨a, (t_eq : s a = t)⟩, (h : x ∈ t)⟩⟩, ⟨a, t_eq.symm ▸ h⟩,
   λ ⟨a, h⟩, ⟨s a, ⟨⟨a, rfl⟩, h⟩⟩⟩
 
+theorem mem_Union_Union {x : γ} {s : α → β → set γ} : x ∈ (⋃ a b, s a b) ↔ ∃ a b, x ∈ s a b :=
```

Comment:
They should be dependent and maybe called `mem_bbUnion`?
```suggestion
theorem mem_bbUnion {β : α → Type*} {s : Π a, β a → set γ} {x : γ} : x ∈ (⋃ a b, s a b) ↔ ∃ a b, x ∈ s a b :=```

Reply:
I think the original name is better, this isn't a `bUnion` as it has no Prop hypothesis.

Another possibility might be `mem_Inter₂` to match some of the `forall₂`-style names

Reply:
Oh right. I misparsed. I like `mem_Union₂`. Actually, this is strictly more general than `mem_bUnion`, right?

Reply:
To illustrate my point about dependence: #11490

Reply:
Now that these theorems take in dependent families, is it even worth having `mem_bUnion_iff` and `mem_bInter_iff`?